rest-parameters.md

```diff
@@ -0,0 +1,64 @@
+# ES6 Rest parameters
+
+Functions can take a rest parameters that act as a placeholder for a non-finite number of arguments.
+
+```javascript
+function score(user, ...marks) {
```

Comment:
I like this example, but it feels out of place at his location. In the sentence above you are highlighting the `non-finite` part of the rest parameter. Let's have the code block detail that.

```js
function logArgsLength(...args) {
    console.log(args.length);
}

logArgsLength(1, 2); // 2
logArgsLength(1, 'a', 'b', 'c', 2, 3) // 6
```